zone/npc.h

```diff
@@ -448,6 +448,7 @@ class NPC : public Mob
 	const char* GetRawNPCTypeName() const { return NPCTypedata_ours ? NPCTypedata_ours->name : NPCTypedata->name; }
 
 	virtual int GetKillExpMod() const { return NPCTypedata_ours ? NPCTypedata_ours->exp_mod : NPCTypedata->exp_mod; }
+	int GetHeroicStrikethrough() const { return NPCTypedata_ours ? NPCTypedata_ours->heroic_strikethrough : NPCTypedata->heroic_strikethrough; }
```

Comment:
Some how I missed this when I initially reviewed it, but this function shouldn't be necessary since we will be using the duped/customized NPCType struct when we initialize the Mob and copy the data to Mob class. If we didn't do that, my initial comment about virtual would have been wrong too :P

Reply:
Thanks! taken care of.